public/app/plugins/panel/stat/README.md

```diff
@@ -1,9 +1,18 @@
-# Singlestat Panel - Native Plugin
+# Stat panel - Native Plugin
 
-The Singlestat Panel is **included** with Grafana.
+The Stat panel is **included** with Grafana.
 
-The Singlestat Panel allows you to show the one main summary stat of a SINGLE series. It reduces the series into a single number (by looking at the max, min, average, or sum of values in the series). Singlestat also provides thresholds to color the stat or the Panel background. It can also translate the single number into a text value, and show a sparkline summary of the series.
+The Stat panel visualization shows a one large stat value with an optional graph sparkline. You can control the background or value color using thresholds.
```

Comment:
`one large statistic`, `The background or value color can be controlled`

Reply:
Fixed.